sdk/Sdk/Targets/Microsoft.Azure.Functions.Worker.Sdk.targets

```diff
@@ -48,7 +48,9 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <FunctionsEnablePlaceholder Condition="'$(FunctionsEnablePlaceholder)' == ''">false</FunctionsEnablePlaceholder>
     <FunctionsEnableWorkerIndexing Condition="'$(FunctionsEnableWorkerIndexing)' == ''">true</FunctionsEnableWorkerIndexing>
     <FunctionsEnableMetadataSourceGen>$(FunctionsEnableWorkerIndexing)</FunctionsEnableMetadataSourceGen>
-    <FunctionsAutoRegisterGeneratedMetadataProvider>$(FunctionsEnableWorkerIndexing)</FunctionsAutoRegisterGeneratedMetadataProvider> 
+    <FunctionsAutoRegisterGeneratedMetadataProvider>$(FunctionsEnableWorkerIndexing)</FunctionsAutoRegisterGeneratedMetadataProvider>
+    <_WriteFunctionsMetadata>true</_WriteFunctionsMetadata>
+    <_WriteFunctionsMetadata Condition="'$(FunctionsEnableMetadataSourceGen)' == 'true'">false</_WriteFunctionsMetadata>
```

Comment:
Do we also want to give users the ability to override this behavior? I don't have a concrete scenario in mind, but if users have a dependency on that file for anything, having a property they can set would help preserve this behavior until the next major version, where we can permanently remove it.

Reply:
What should our default be? Write or don't write?

Reply:
Went with default `false`.